.github/workflows/test.yml

```diff
@@ -0,0 +1,34 @@
+name: Django Thumbnails
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    services:
+      redis:
+        image: redis
+        options: >-
+          --health-cmd "redis-cli ping"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 6379:6379
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
```

Comment:
can you maintain the build matrix from travis?
https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/managing-complex-workflows#using-a-build-matrix

https://github.com/ui/django-thumbnails/blob/fa24c15e9620244e201e403192e3b18ce8106f41/.travis.yml#L3-L11